### ISSUE TITLE
Extraction Not Working

### ISSUE THREAD

**Issue by MKan21** [cadon/ARKStatsExtractor#955]
I tried extracting stat distributions for a couple critters but none are working out. I dragged the ini settings into the extractor settings, and I am using exported dino data for the stat inputs. Because of this, I do not think its a user input problem. How should I proceed with fixing this? Thanks in advanced for any help :)

**Comment by VolatilePulse** [comment]
Do you have some screenshots as to what's going on? You didn't mention if it was a failed extraction or if you're just having issues on how to use the extractor.

**Comment by MKan21** [comment]
The extractor reports incorrect data for stat point distribution and then offers solutions such as input error, incorrect server settings and so on.

**Comment by VolatilePulse** [comment]
Screenshots help quite a bit, but you can also provide the export files that are causing you problems as well as the configurations you are using and if your server is using any mods. Unfortunately, it is hard to troubleshoot without any information to work with. The extractor does have a couple of issues, but none that most users will ever experience.

**Comment by MKan21** [comment]
![Screenshot (20)](https://user-images.githubusercontent.com/55668199/65712839-c2180080-e04c-11e9-8313-5d3ab6110ff2.png)

![Screenshot (21)](https://user-images.githubusercontent.com/55668199/65712899-db20b180-e04c-11e9-8890-f7d2cbaab046.png)

![Screenshot (23)](https://user-images.githubusercontent.com/55668199/65713018-16bb7b80-e04d-11e9-8d9f-c09c96bf649f.png)

![Screenshot (24)](https://user-images.githubusercontent.com/55668199/65713023-191dd580-e04d-11e9-9ab1-3932c089f1cd.png)

![Screenshot (25)](https://user-images.githubusercontent.com/55668199/65713046-23d86a80-e04d-11e9-8d63-ee794a341519.png)

![Screenshot (22)](https://user-images.githubusercontent.com/55668199/65713064-2dfa6900-e04d-11e9-857a-c31344641792.png)
 My active mods are: 1272830262,895711211,478736056,1184763409,849372965,1359946021,671591278,821530042,1171853632,522160582,1387191206,889745138,854186603,937322940,1086678105,947524058

**Comment by cadon** [comment]
Did you change any of the stat-multipliers recently?
It looks like setting the TameAdd multiplier for HP to 1.6 instead of 0.45 gives better results, but it's not quite solved with changing that.
It could be that the Rex shows outdated values which were calculated with outdated stat-multipliers.
Singleplayers are disabled in ARK?
After looking through the mods, the only mod that should matter for ASB is classic flyers, but that doens't affect the Rex at all. You should add this mod in ASB though if you want to use it for flyers.

**Comment by MKan21** [comment]
Ill try messing around with the TameAdd multiplier.
I tamed these Rexs for the purpose of testing out the Breeding calculator, within minutes of downloading it. I'm not thinking they're outdated, but I am also not entirely sure how that works. 
Its a non dedicated server, how do you mean "Singleplayers are disabled"?

**Comment by VolatilePulse** [comment]
I believe he meant the Single player setting on your server. It's under one of the config pages you posted earlier. If SinglePlayer settings are disabled, they need to be in ASB as well, otherwise, you need to make sure that check box is checked as it does affect creature stat values.

**Comment by MKan21** [comment]
I do have single player settings enabled on both now, but I am still getting a red box for health on the extractor.

**Comment by VolatilePulse** [comment]
So, I'm going to follow up on the stat values again to make sure I didn't miss anything, but this mod ([Dino Station!](https://steamcommunity.com/sharedfiles/filedetails/?id=1272830262)) is likely the issue. It allows you to artificially add levels to creatures, circumventing the required calculations ASB requires.

If you use this mod on a creature, the creature can no longer be used in ASB as it's levels will no calculate properly any more. The only way to continue to use this mod and ASB together would be to use the Save File extraction and even that I'm not 100% confident will work.

**Comment by VolatilePulse** [comment]
Well, that Rex does actually extract fine for me. Looks like it was freshly tamed, however, I would recommend considering my warning about the Dino station mod. As for the Rex, the only thing I noticed was that the Taming Effectiveness would have been rather low, lower than the standard 80-100% default. Otherwise, no settings were changed.

![image](https://user-images.githubusercontent.com/6995232/65826532-d3cbf480-e254-11e9-8aa7-268b5d0ba3a8.png)
